.changeset/small-crabs-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@alephium/mobile-wallet": minor
+---
+
+Dashboard and lists redesign, new dedicated NFT screen
```

Comment:
nit: It'd be better if this is split into 2 changesets :)

Reply:
aaaah I knew :) but will it bump the version twice?

Reply:
No, it won't!